### ISSUE TITLE
The fork date should be postponed

### ISSUE THREAD

**Issue by ethan-nx** [BTCGPU/BTCGPU#66]
Clearly with so many work-in-progress features, including the one perceived as mandatory for most supporters - 2-way replay protection - it seems most reasonable to postpone the fork until:

- you have a 2-way replay protection implemented so that users can easily split their coins,
- you have a testnet developers can play with,
- you have presented exact way the premine and timelocks will be executed,
- you have a client people could actually run,
- there is no block explorer.

Right now there are only a few hours till the fork.

Without replay protection BTG loses most of it's value since Bitcoin holders will not be able to send BTG coins without the risk of losing their BTC coins and vice versa.

Moreover, lack of replay protection makes it really difficult for services like exchanges to credit their users with BTG coins.

If you really want BTG to have any value for its holders you should definitely postpone the launch until all the issues are resolved. Otherwise you risk jeopardizing your whole project.

Cheers,
Ethan (from YOLOdice)

**Comment by CaptJakk** [comment]
Yeah I'm generally inclined to agree here. It looks like the build is failing 5 hours before the chain is supposed to go live. There isn't really any clear docs on how it should be mined. The only information is that it will be using equihash, but it isn't clear whether or not we can just repurpose Zcash mining software to do so.

**Comment by adhood** [comment]
There has been a lot of misinformation going around.

The "fork" is simply going to be when a snapshot of the Bitcoin blockchain is taken. The testnet and mainnet and everything else that's needed to make Bitcoin Gold "work" is still a work in progress and will not be going live at that time. The ONLY thing anyone will be able to do is trade Bitcoin Gold on exchanges that support it.

You will not be able to mine, send or receive Bitcoin Gold until the mainnet is actually created.

See #39 -- As of today, it seems we are just waiting for the Bitcoin Gold team to reopen the bounty ticket so that whoever wants to code the testnet and mainnet can get started.

**Comment by barrystyle** [comment]
@CaptJakk the build doesnt fail? however it doesnt 'sync' either.. as i understood it should be able to sync from any bitcoin node, and will changeover at block 491,407

**Comment by vaverix** [comment]
As I understand correctly, the *actual* fork will be after 1st November when everything will be completed. 25 October is the date of network snapshot and nothing more.

**Comment by barrystyle** [comment]
@adhood It's a fork of Bitcoin mainnet; sync up to block 491,407..

**Comment by gau27** [comment]
Even with that said, I feel a week is a tight timeline for implementing,testing and debugging. Code is made airtight and battle ready over time after a draft has been written. My 2 cents.

**Comment by TheComputerGenie** [comment]
> If I understand correctly, the actual fork will be after 1st November when everything will be completed. 

You don't. The very tick that consensus rules are ignored and a different path is taken, a "fork" is created. If one allows the notion that anything done outside the consensus rules is actually valid, then the "fork" becomes recorded with he 1st premined block. And then **another** "fork" will occur reinstating consensus rules (but **not** allowing consensus to invalidate the non-consensus premine).
